lib/rubocop/cop/style/negated_if_else_condition.rb

```diff
@@ -35,6 +35,8 @@ class NegatedIfElseCondition < Base
 
         NEGATED_EQUALITY_METHODS = %i[!= !~].freeze
 
+        def_node_matcher :double_negative?, '(send (send _ :!) :!)'
```

Comment:
Let's name this `double_negation?` to be consistent with the terminology we've used so far.

Reply:
Sure! I updated it.